src/compositions/Login/LoginComponent.js

```diff
@@ -26,7 +28,7 @@ class LoginComponent extends Component {
 
     handleLoggingUser = () => {
         login(this.state).then(user => {
-            if (user._id) {
+            if (user[0]._id) {
```

Comment:
Might need a safety check here.  It'll crash (and not show the error message) if you get a non-array from an error.
```js
if (user[0] && user[0]._id) {
    // ...
}
```